src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import { User } from '../../types/types';
+import './UserInfo.scss';
+
+export type Props = {
```

Comment:
Consider giving a more descriptive name for the type, like `UserInfoProps` instead of `Props`. This can make it easier to understand the code when consumed by other components 😊